presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -0,0 +1,200 @@
+=========================
+Noisy Aggregate Functions
+=========================
+
+Noisy aggregate functions are aggregate functions that provide random, noisy
+approximations of common aggregations like :func:`sum`, :func:`count`, and
+:func:`approx_distinct` as well as sketches like :func:`approx_set`. By
+injecting random noise into results, noisy aggregation functions make it more
+difficult to determine or confirm the exact data that was aggregated.
+
+.. note::
+
+    While many of these functions resemble
+    `differential privacy <https://en.wikipedia.org/wiki/Differential_privacy>`_
+    mechanisms, it is important to note that neither the values returned by
+    these functions nor the query results that incorporate these functions
+    are differentially private in general. See `Limitations`_ below for more
+    details. Users who wish to support a strong privacy guarantee should
+    discuss with a suitable technical expert first.
+
+Counts, Sums, and Averages
+--------------------------
+
+.. function:: noisy_count_gaussian(x, noise_scale[, random_seed]) -> bigint
+
+    Counts the non-``NULL`` values in ``x`` and then adds random Gaussian noise with 0
+    mean and standard deviation of ``noise_scale`` to the true count. The noisy count
+    is post-processed to be non-negative and rounded to bigint.
+
+    If provided, ``random_seed`` is used to seed the random number generator. Otherwise,
+    noise is drawn from a secure random. ::
+
+        SELECT noisy_count_gaussian(orderkey, 20.0) FROM tpch.tiny.lineitem; -- 60179 (1 row)
+        SELECT noisy_count_gaussian(orderkey, 20.0) FROM tpch.tiny.lineitem WHERE false; -- NULL (1 row)
+
+.. note::
+
+    Unlike :func:`count`, this function will return ``NULL`` when the (true) count of ``x`` is 0.
+
+    Distinct counting can be performed using ``noisy_count_gaussian(DISTINCT x, ...)``, or with
+    :func:`noisy_approx_distinct_sfm`. Generally speaking, :func:`noisy_count_gaussian` will
+    return more accurate results but at a larger computational cost.
+
+.. function:: noisy_count_if_gaussian(x, noise_scale[, random_seed]) -> bigint
+
+    Counts the ``TRUE`` values in ``x`` and then adds a random Gaussian noise with 0 mean
```

Comment:
a random --> random

This is a single value. I'm not sure if that's technically noise or not. I suppose it would be accurate to say "adds a normally distributed random double value  with 0 mean..." instead

Reply:
I think your suggestion is clearer, thanks!